EthOn.rdf

```diff
@@ -1691,9 +1691,9 @@
     <owl:Class rdf:about="http://ethon.consensys.net/AccountStorage">
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/AccountConcept"/>
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/ModifiedMerklePatriciaTree"/>
-        <simpleDefinition>A data structure which contains the current state of a smart contract defined in a contract account, i.e. the total supply of an ERC20 token and the mapping of account addresses to current balances.</simpleDefinition>
+        <simpleDefinition>A data structure which contains the current storage state of a smart contract, i.e. the total supply of an ERC20 token and the mapping of account addresses to current balances.</simpleDefinition>
         <suggestedStringRepresentation xml:lang="en">AccountStorage</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">A Merkle Patricia tree that encodes the storage contents of an account. It is not used to store an account&apos;s code, but the execution state of the code. The account&apos;s EVM code is stored in the account state.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The storage state of a contract account contains the values of all variables defined in the contract's code at a certain point in time. Usually Ethereum clients implement this structure as a Merkle Patricia Tree. Whenever any of the contract's variables are changed a new state is created.</rdfs:comment>
```

Comment:
Could you explain to me the difference between `AccountStorage` and `AccountState`? When you begin with "The storage state of a contract..." it is a bit confusing because 'storage state' seems like it could refer to either of these concepts.